requirements-dev.txt

```diff
@@ -5,8 +5,8 @@ pre-commit
 pytest-cov
 
 # TODO pin exact version for pylint and astroid
```

Comment:
Can resolve this TODO now

Reply:
and be sure to remove the comment 😁

Reply:
the comment has been here for 2 years! it will be gone now